### ISSUE TITLE
Build showcase notebook

### ISSUE THREAD

**Issue by ArthRie** [ArthRie/trimkmeans#1]
Build a notebook for presentations etc. containing test data, benchmarking, visualiztation and comparison to r trimkmeans function and pythons regular kmeans

- [x] #1
- [x] create notebook
- [x] create test data in notebook
- [x] time and plot the runtime of the implementation and the r original with 4 runs of varying size
- [x] visualize clusters with trimkmeans next to a sklearn.kmeans run next to r original
- [x] split performance metrics and runtime evaluation into two notebooks
- [x] sklearn.metrics.silhouette_score compare to untrimmed kmeans on multiple examples with varying datasize and n_clusters
- [x] try creating sample data where sklearn.metrics.adjusted_rand_score gets better with trimmed?
- [x] test one additional metric
- [x] run on two more 2d shapes 
- [x] run on two 3d shapes
- [x] refactor 
- [x] four more 2D visualizations from https://scikit-learn.org/stable/auto_examples/cluster/plot_kmeans_assumptions.html
- [x] plot comparison with r on predefined centroids
